tmc.py

```diff
@@ -0,0 +1,159 @@
+import numpy as np
```

Comment:
Why is this file being included? It looks like a copy of mc.py?

Reply:
It was a backup of mc.py while I was working on that. It has been removed from the repo.